styles.css

```diff
@@ -287,3 +287,68 @@ header {
 .portfolio-details h4 {
   font-size: 24px;
 }
+
+#contact-form {
+  background: url(assets/contact-frame.svg);
+  background-repeat: no-repeat;
+  background-position: right bottom;
+}
+
+form {
+  padding-top: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 10px;
+  align-items: center;
+  font-family: 'Inter', sans-serif;
+  border-radius: 4px;
+  font-weight: 400;
+  width: 100%;
+}
```

Comment:
- [ ] As I mentioned in my above `inline-comment` related to aligning your `form` item using the **flexbox** and I can see you did an awesome job by implementing the `flexbox` for your form container but the problem is your design is still `not` **matching** the original *Figma* design. Kindly take a look at the below design comparison table to understand the problem, and also you will find the bullet points for the improvements.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-07-12 00-47-11](https://github.com/AbdulSalamAbro/portfoliowebsite/assets/66476329/3186d929-b060-48b6-bb77-f5c13405dc0e)

</td>
<td>

![Screenshot from 2023-07-12 00-42-24](https://github.com/AbdulSalamAbro/portfoliowebsite/assets/66476329/2f6dc722-0eee-4be5-a442-11eb7f1adfc7)

</td>
</tr>
</table>

### Improvements 📝 

1. Make sure that you are using the correct `border` colour, which in your case is not ok. Your border colour should match the `Figma` design.
2. Your `inputs` or `textarea` elements are not aligned with each other properly, so please align them and look for the issue, most probably I already mentioned the things above, but still cross-check and find the root cause of the alignment problem.
3. Please use the `appropriate` spacing between each `form` child element and also you need to provide the margin-bottom spacing to have a good space between your submit button and email.
4. Kindly use the `%age` over the fixed absolute units like `px` to make your design looks responsive right now the problem is your design is not responsive and you are not following the `project` requirements properly.
    - #### Video
    [Screencast from 12-07-23 01:09:51 AM IST.webm](https://github.com/AbdulSalamAbro/portfoliowebsite/assets/66476329/9503c18e-e736-49e5-9697-6a1edcb19e36)

   - #### Requirement
   ![image](https://github.com/AbdulSalamAbro/portfoliowebsite/assets/66476329/0143e915-3cfe-42af-b8a1-79fc46a32bf7)

5. Your contact form text is also not aligned to the form, so please align the text properly, even though you can see the things in above attached video. But you can take a look at the below screenshot.
    ![image](https://github.com/AbdulSalamAbro/portfoliowebsite/assets/66476329/0bfdea34-1956-4996-b2b0-bea1e10379b2)

Reply:
> * [ ]  As I mentioned in my above `inline-comment` related to aligning your `form` item using the **flexbox** and I can see you did an awesome job by implementing the `flexbox` for your form container but the problem is your design is still `not` **matching** the original _Figma_ design. Kindly take a look at the below design comparison table to understand the problem, and also you will find the bullet points for the improvements.
> 
> ### Requirement
> ![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)
> 
> ### Design Comparison
> Figma Design	Your Design
> ![Screenshot from 2023-07-12 00-47-11](https://user-images.githubusercontent.com/66476329/252773072-3186d929-b060-48b6-bb77-f5c13405dc0e.png)
> 
> ![Screenshot from 2023-07-12 00-42-24](https://user-images.githubusercontent.com/66476329/252773051-2f6dc722-0eee-4be5-a442-11eb7f1adfc7.png)
> 
> ### Improvements 📝
> 1. Make sure that you are using the correct `border` colour, which in your case is not ok. Your border colour should match the `Figma` design.
> 2. Your `inputs` or `textarea` elements are not aligned with each other properly, so please align them and look for the issue, most probably I already mentioned the things above, but still cross-check and find the root cause of the alignment problem.
> 3. Please use the `appropriate` spacing between each `form` child element and also you need to provide the margin-bottom spacing to have a good space between your submit button and email.
> 4. Kindly use the `%age` over the fixed absolute units like `px` to make your design looks responsive right now the problem is your design is not responsive and you are not following the `project` requirements properly.
>    
>    * #### Video
>    
>    
>      
>        
>        
>    
>        Screencast.from.12-07-23.01.09.51.AM.IST.webm
>        
>      
>    
>      
>    
>      
>    
>    
>    
>    * #### Requirement
>    
>    ![image](https://user-images.githubusercontent.com/66476329/252774726-0143e915-3cfe-42af-b8a1-79fc46a32bf7.png)
> 5. Your contact form text is also not aligned to the form, so please align the text properly, even though you can see the things in above attached video. But you can take a look at the below screenshot.
>    ![image](https://user-images.githubusercontent.com/66476329/252775436-0bfdea34-1956-4996-b2b0-bea1e10379b2.png)

- [ ] Nice one fixing some of the reviewer's suggestions, kindly note that you didn't fix any of the above suggestions, kindly fix them to ensure you are attentive to details and open to learn, which are outstanding qualities for your future job 🤗.

Reply:
everything is updated now and i also used percentage now for aligning the input and text field hope you can see and will satisfied with your required changes thanks